### ISSUE TITLE
vm/dart_2/use_strip_flag_test used to be flaky, now consistently timing out on tsan bot

### ISSUE THREAD

**Issue by aam** [dart-lang/sdk#42986]
https://ci.chromium.org/p/dart/builders/ci.sandbox/vm-kernel-precomp-tsan-linux-release-x64/884 is seemingly when the test lost its flakiness.

**Comment by whesse** [comment]
Yes, the test went from flaky to a failing status.  Looking at the previous approval, the new test was approved as timing out because it supposedly went to passing while it was in flaky status.  But now it is a stable timeout when it becomes stable again, so nobody has ever filed an issue or approved it being in a timeout status.
